app/components/tags.tsx

```diff
@@ -0,0 +1,21 @@
+import type {Tag as TagType} from '~/server-utils/stampy'
+
+type Props = {
+  tags: TagType[]
+  [k: string]: any
+}
+
+export default function Tags({tags}: Props) {
+  return (
+    <div className="tags-container">
+      <div>
```

Comment:
this extra div is because I don't do CSS - my solution to having the tags and actions on one line is:
![untitled](https://user-images.githubusercontent.com/3942390/217357217-57dbf935-20a2-41e5-9f96-38e52cfb2036.png)


Please tell me there's a better way...